client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.controller.js

```diff
@@ -169,9 +162,9 @@ angular.module("App").controller(
         migrateAccount () {
             this.loaders.isWaitingForMigration = true;
 
-            this.Emails.migrateAccountToDestinationAccount(this.email.domain, this.email.accountName, this.migrate.destinationService, this.migrate.destinationEmail, this.migrate.password)
+            this.Emails.migrateAccountToDestinationAccount(this.email.domain, this.email.accountName, this.migrate.destinationService.name, this.migrate.destinationEmail, this.migrate.password)
                 .then(() => {
-                    if (this.migrate.serviceType === "EMAIL PRO") {
+                    if (this.migrate.destinationService.type === "EMAIL PRO") {
```

Comment:
Right above you're using `_.get(this, "migrate.destinationService.type")` and it feels safer :)

Reply:
Yes, done